packages/xo-web/src/common/report-bug-button.js

```diff
@@ -8,9 +8,13 @@ import ActionRowButton from './action-row-button'
 export const CAN_REPORT_BUG = __DEV__ && process.env.XOA_PLAN > 1
 
 export const reportBug = ({ formatMessage, message, title }) => {
-  const encodedTitle = encodeURIComponent(title)
+  const encodedTitle = encodeURIComponent(title == null ? '' : title)
   const encodedMessage = encodeURIComponent(
-    formatMessage !== undefined ? formatMessage(message) : message
+    formatMessage !== undefined
```

Comment:
Check `message` before `formatMessage` otherwise one of your branch is incorrect.

Reply:
But we still want to be able to handle the `undefined` case in `formatMessage`. Then, `formatMessage` should make sure not to return `undefined`.

Reply:
Will change it anyway.